File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        String A = String.valueOf(a);
        String B = String.valueOf(b);
        int anum = Integer.valueOf(A+B);
        int bnum = Integer.valueOf(B+A);
        if(anum>bnum){
            return anum;
        }
        else{
            return bnum;
        }
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        String A = Convert.ToString(a);
        String B = Convert.ToString(b);
        int anum = int.Parse(A+B);
        int bnum = int.Parse(B+A);
        if(anum>bnum){
            return anum;
        }
        else{
            return bnum;
        }
    }
}